tests/mu_expand_no_expand_test.c

```diff
@@ -9,6 +9,11 @@
 
 #include "mu_expand_no_expand_test.h"
 
+// For gcc build that causes a false negative
+#ifndef _MSC_VER
+#pragma GCC diagnostic ignored "-Wunused-value"
```

Comment:
did you fix this with the "(void)variable_name" in the other file? Just wondering if it's an easy fix to be explicit vs. disabling the warning. /consider

Reply:
The unused variable in this file is in a macro that gets constructed so I couldn't just add a void to it.